packages/block-editor/src/components/inserter/menu.js

```diff
@@ -45,6 +46,7 @@ function InserterMenu(
 		showMostUsedBlocks,
 		__experimentalFilterValue = '',
 		shouldFocusBlock = true,
+		__experimentalOnPatternCategorySelection = NOOP,
```

Comment:
This seems like a new public API. Just a reminder to stop adding prefixed experimental APIs like that. It's preferred to add private APIs going forward. Thanks.

Reply:
Yes, good point! 🤦🏻 I copied https://github.com/WordPress/gutenberg/pull/56773 verbatim. Would a linter that says no when we use `__experimental` help ... probably not 😁

Reply:
Oh yes!

Reply:
Opened #62110 but I am a bit unclear how to proceed with privatizing a component that was public but marked experimental - since that will create a public experimental and a private experimental component :/

Reply:
Alternative PR here: https://github.com/WordPress/gutenberg/pull/62130. Would love some testing to land this for 6.6.